src/background/messages/fetchCurrentProject.test.ts

```diff
@@ -0,0 +1,34 @@
+import { describe, expect, it, jest } from "@jest/globals"
+import fetchMock from "jest-fetch-mock"
+
+import type { PlasmoMessaging } from "@plasmohq/messaging"
+
+import * as testData from "../../../test-data.json"
+import handler from "./fetchCurrentProject"
+
+fetchMock.enableMocks()
+
+describe("fetchCurrentProject", () => {
+  beforeEach(() => {
```

Comment:
My IDE is saying that the import call is missing for beforeEach, which is strange as the tests still pass without it. But it stops complaining when you add it the the @jest/globals import statement.